CMakeLists.txt

```diff
@@ -319,10 +321,22 @@ set(CPACK_DEBIAN_PACKAGE_DEPENDS "${IRODS_PACKAGE_DEPENDENCIES_STRING},
     irods-runtime (= ${IRODS_CPACK_DEBIAN_PACKAGE_VERSION}), libc6")
 
 if (IRODS_LINUX_DISTRIBUTION_NAME STREQUAL "ubuntu")
-    if (IRODS_LINUX_DISTRIBUTION_VERSION_MAJOR GREATER_EQUAL "18")
-         set(CPACK_DEBIAN_PACKAGE_DEPENDS "${CPACK_DEBIAN_PACKAGE_DEPENDS}, libnetcdf13, libhdf5-100")
+    if (IRODS_LINUX_DISTRIBUTION_VERSION_MAJOR GREATER_EQUAL "22")
+        set(CPACK_DEBIAN_PACKAGE_DEPENDS "${CPACK_DEBIAN_PACKAGE_DEPENDS}, libnetcdf19, libhdf5-103")
```

Comment:
Even though we do not claim support and it is not widely used, would these be the right versions of these packages on, say, Ubuntu 23? In other words, should we be a little more strict about these OS version numbers?

Reply:
Absolutely. Let's lock things down to the OS which are supported. That way the dependencies are documented in the project and there aren't any surprises when we start adding support for new OS.

Reply:
I think I've now done this correctly.  Tested it, even, and an error occurs if not building it on a supported OS.

< RSVP >

Reply:
Looks good to me. Please resolve this conversation if happy with it.